test/unit/profile_test.rb

```diff
@@ -69,14 +69,13 @@
       let(:profile_id) { 'empty-metadata' }
 
       it 'prints loads of warnings' do
-        inspec_yml = "#{home}/mock/profiles/#{profile_id}/inspec.yml"
         logger.expect :info, nil, ["Checking profile in #{home}/mock/profiles/#{profile_id}"]
-        logger.expect :error, nil, ["Missing profile name in #{inspec_yml}"]
-        logger.expect :error, nil, ["Missing profile version in #{inspec_yml}"]
-        logger.expect :warn, nil, ["Missing profile title in #{inspec_yml}"]
-        logger.expect :warn, nil, ["Missing profile summary in #{inspec_yml}"]
-        logger.expect :warn, nil, ["Missing profile maintainer in #{inspec_yml}"]
-        logger.expect :warn, nil, ["Missing profile copyright in #{inspec_yml}"]
+        logger.expect :error, nil, ["Missing profile name in inspec.yml"]
```

Comment:
Mhm this is much better. Together, line 72 and 73 still give you all the information...